spec/units/token_parser_spec.rb

```diff
@@ -173,6 +173,17 @@
         env: env, command: 'commit', args: [],
       )
     end
+
+    it 'parses blank lines' do
+      expect(parse(tokens([:EOS]))).to be_a Gitsh::Commands::Noop
+      expect(parse(tokens([:SPACE], [:EOS]))).to be_a Gitsh::Commands::Noop
+    end
+
+    it 'does not parse just a semicolon' do
```

Comment:
Although I suppose it's complicated by the fact that a noop usually means executing a default command, so maybe this is the right approach.